ui/app/styles/components/recovery.scss

```diff
@@ -0,0 +1,17 @@
+@use '../utils/size_variables';
+
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+.snapshot-card-container {
+  padding: size_variables.$spacing-12 size_variables.$spacing-24;
+  margin-bottom: size_variables.$spacing-24;
+  width: fit-content;
+
+  .flex-row {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
We have flex styling [here](https://github.com/hashicorp/vault/blob/main/ui/app/styles/helper-classes/flexbox-and-grid.scss#L15) that could be used instead of custom css 

`flex space-between` should be the same as this?